test/runtests.jl

```diff
@@ -213,6 +213,18 @@ float_types = (Float32, Float64)
             end
         end
 
+        @testset "Divergence-free solution [GPU]" begin
+            println("    Testing divergence-free solution [GPU]...")
+            @hascuda begin
+                for ft in float_types
```

Comment:
btw, It's an almost-universally adopted julia convention to denote a 'type' with capital letters (`T`, `FT`, etc).

Reply:
True. I'll use `FT` in the future, and modify `runtests.jl` when I get the chance.